hooks/50-tmpfiles-d.chroot

```diff
@@ -0,0 +1,25 @@
+#!/bin/sh
+
+set -e
+
```

Comment:
Can you please give a brief comment documenting how this works, ideally with a URL to full systemd docs on the topic please.